src/store/components/AdminQuantitiesPage/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import { PublicMerchCollection } from '../../../types';
+
+import StoreHeader from '../StoreHeader';
+
+import './style.less';
+
+interface AdminQuantitiesPageProps {
+  collections: PublicMerchCollection[] | undefined;
+}
+
+const AdminQuantitiesPage: React.FC<AdminQuantitiesPageProps> = (props) => {
+  const { collections } = props;
+
+  return (
+    <>
+      <StoreHeader breadcrumb breadcrumbLocation="/store/admin" />
+      <div className="admin-quantities-page">
+        <h2 className="admin-quantities-page-title">Item Quantities</h2>
+        <div>
+          {collections &&
+            collections?.map((collection) => {
+              return (
+                <div className="admin-quantities-page-collection" key={collection.uuid}>
+                  <p className="admin-quantities-page-collection-title">{collection.title}</p>
+                  {collection.items
+                    .sort((a, b) => {
+                      const textA = a.itemName.toUpperCase();
+                      const textB = b.itemName.toUpperCase();
+                      if (textA < textB) {
+                        return -1;
+                      }
+                      if (textA > textB) {
+                        return 1;
+                      }
+                      return 0;
+                    })
+                    .map((item) => {
+                      return (
+                        <div className="admin-quantities-page-item" key={item.uuid}>
+                          <p className="admin-quantities-page-item-title">{item.itemName}</p>
+                          <p>{item.options.length === 1 && item.options[0].quantity}</p>
+                          <div className="admin-quantities-page-item-options">
+                            {item.options.length !== 1 &&
+                              item.options
+                                .sort((a, b) => {
+                                  if ((a.metadata?.position ?? 0) < (b.metadata?.position ?? 0)) {
+                                    return -1;
+                                  }
+                                  if ((a.metadata?.position ?? 0) > (b.metadata?.position ?? 0)) {
+                                    return 1;
+                                  }
+                                  return 0;
```

Comment:
not sure if `position` is a number but I think `a.position - b.position` would suffice?

Reply:
Yep, I guess I didn't understand `.sort()` well enough before implementing